manifests/registry/base/postgres-keycloak.yaml

```diff
@@ -0,0 +1,107 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgres-keycloak
+spec:
+  type: ClusterIP
+  ports:
+  - name: pg-keycloak
+    port: 5432
+    protocol: TCP
+    targetPort: pg-keycloak
+  selector:
+    statefulset: postgres-keycloak
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-keycloak
+type: Opaque
+data:
+  postgres-password: YV9iZXR0ZXJfcGFzc3dvcmRfdGhhbl90aGlz
+  #postgres-password: a_better_password_than_this
+            
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: postgres-keycloak-claim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Mi 
+      
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: postgres-keycloak
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      statefulset: postgres-keycloak
+  serviceName: postgres-keycloak
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        statefulset: postgres-keycloak
+    spec:
+      containers:
+      - name: postgres
+        # image: postgres:11.1
+        image: quay.io/centos7/postgresql-12-centos7:1
+        env:
+        - name: POSTGRESQL_ADMIN_PASSWORD
+          value: admin1234
```

Comment:
maybe extract this to a secret like you're doing with the regular PostgreSQL password?